contrib/grafana-watcher/main.go

```diff
@@ -16,31 +16,67 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"net/url"
 	"os"
 	"path/filepath"
+	"strings"
 	"time"
 
 	fsnotify "gopkg.in/fsnotify.v1"
 
 	"github.com/coreos/prometheus-operator/contrib/grafana-watcher/grafana"
 	"github.com/coreos/prometheus-operator/contrib/grafana-watcher/updater"
+	"github.com/spf13/pflag"
 )
 
-var (
-	watchDir   = flag.String("watch-dir", "", "The directory the ConfigMap is mounted into to watch for updates.")
-	grafanaUrl = flag.String("grafana-url", "", "The url to issue requests to update dashboards to.")
-)
+type watchDirSet map[string]struct{}
+
+func (w *watchDirSet) String() string {
+	s := *w
+	return strings.Join(s.asSlice(), ",")
+}
+
+func (w *watchDirSet) Set(value string) error {
+	s := *w
+	cols := strings.Split(value, ",")
```

Comment:
I'm not a huge fan of introducing both `--flag=a --flag=b,c`, let's just stick with a repeatable flag `--flag=a --flag=b --flag=c`.